efel/tests/test_cppcore.py

```diff
@@ -154,3 +154,22 @@ def test_getFeature_non_existant(self):
         """cppcore: Testing failure exit code in getFeature"""
         import efel.cppcore
         efel.cppcore.getFeature("does_not_exist", list())
+
+    def test_registerFeature(self):
+        import efel.cppcore
+
+        def getTwiceAP_Amp(output):
+            feature_values = list()
+            efel.cppcore.getFeature('AP_amplitude', feature_values)
+            output.extend(2*x for x in feature_values)
+            return len(feature_values)
+
+        self.setup_data()
+        efel.cppcore.registerFeature('getTwiceAP_Amp', getTwiceAP_Amp)
```

Comment:
Ah, ok.

Then, the idea would be that features can still depend on the old version if that's what their dependency says?  That means we have to version all features, including the ones that already exist, right?